release.yaml

```diff
@@ -52,8 +52,10 @@ rancher-istio:
 - 101.1.0+up1.15.3
 rancher-logging:
 - 101.1.0+up3.17.10
```

Comment:
For some reason your PR has the 101 versions of the chart still in the release yaml. You might have missed [this pr](https://github.com/rancher/charts/pull/2390/files#diff-a0b29149f10adefb5c44814777ca3aa3c992d81fd92a1a051add31fab99452fcL56) in the rebase?

Reply:
The removal of the unreleased version happened after this PR was merged.

Reply:
oops ignore, wrong pr